src/libs/actions/Report.js

```diff
@@ -1314,8 +1314,14 @@ function syncChatAndIOUReports(chatReport, iouReport) {
  * @param {String} notificationPreference
  */
 function updateNotificationPreference(reportID, notificationPreference) {
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {notificationPreference});
-    DeprecatedAPI.Report_UpdateNotificationPreference({reportID, notificationPreference});
+    const optimisticData = [
+        {
+            onyxMethod: 'merge',
```

Comment:
NAB, I'm adding a const for this in another PR but might be good to add a const for all the `'merge'`

Reply:
Mind linking to the PR so I can add the same constant? It'll be easier to fix any conflicts that way.